keps/9270-multikueue-incremental-step-size/kep.yaml

```
@@ -0,0 +1,32 @@
+title: Introduce configuration for the MultiKueue Incremental Dispatcher
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)